### ISSUE TITLE
"Attorney" type is ambiguous, consider "Law Firm" or "Law Office" as another type for the business

### ISSUE THREAD

**Issue by fosterinteractive** [schemaorg/schemaorg#784]
I've been working on a website for a law firm and have discovered some ambiguities associated with  https://schema.org/Attorney

My assumption about the semantic meaning of this type, is that it refers to a Business who's primary service is advising, and representing people or companies in matters related to the law. This business entity could be a single lawyer, or a group of lawyers working together. In addition this business entity could have many non-lawyer employees supporting the business.

But the dictionary terms Attorney is "A person, typically a lawyer, appointed to act for another in business or legal matters."  (http://www.oxforddictionaries.com/definition/english/attorney) or "In the most general sense this term denotes an agent or substitute, or one who is appointed and authorized to act in the place or stead of another. (http://thelawdictionary.org/attorney/)

The distinction is that an attorney is a type of person, not a type of business. 

In my personal experience in Canada and the US there are many examples of single lawyers operating as the sole lawyer in there business. When people talk about people in this structure they are basically operating as a "sole proprietorship" so the person and business are same thing. In this case the Attorney type is fine. It also makes sense that solo lawyers could have employees such as an assistant helping them with the sole-proprietorship style business.

There are also many lawyers who work as part of a team. In Canada (and I'm pretty confident for the USA as well) groups of lawyers are not legally allowed to incorporate a business which shields the business owners from liability lawsuits against the business. They are legally responsible for the advise they give. So groups of lawyers forming a business create a "Partnership" where all the lawyers share responsibility for the actions of the group. This is typically called a "Law Firm".  http://thelawdictionary.org/law-firm/

Basically it's the analogous to a Doctor vs Doctor's office.

**Comment by scor** [comment]
Thanks @fosterinteractive. When we first spoke about this a few weeks ago, you went into a lot more details (feel free to re-iterate in the summary) and you were not sure if the ambiguity was due to a difference of terminology between Canada and the US (and maybe other countries). 

PS: there are also some unfinished statements in your summary: "This business entity"?

**Comment by fosterinteractive** [comment]
(sorry hit submit while typing... I've posted more details here)

**Comment by fosterinteractive** [comment]
One other consideration. Wikipedia shows that a "law firm" can be a single practitioner. https://en.wikipedia.org/wiki/Law_firm while other sources say it must be 2 or more lawyers http://www.businessdictionary.com/definition/law-firm.html . 

I have come across another term that I think is better as it covers all cases with less ambiguity. "Law Office" makes sense for all sizes/structures of business providing legal servers.  http://research.lawyers.com/understanding-how-law-offices-do-business.html

**Comment by danbri** [comment]
Thanks! See also previous discussion - https://lists.w3.org/Archives/Public/public-vocabs/2014Mar/0089.html  /cc @aaranged

**Comment by danbri** [comment]
Ok we have been cleaning up around ProfessionalService (which was confusing a lot of people w.r.t. the entirely different Service type). As part of this, ProfessionalService as a supertype for various things including Attorney is being deprecated. I've just implemented a LegalService type as a supertype of Attorney and Notary, see
http://sdo-phobos.appspot.com/docs/releases.html#service-usability-fixes

http://sdo-phobos.appspot.com/LegalService: "A LegalService is a business that provides legally-oriented services, advice and representation, e.g. law firms. As a LocalBusiness it can be described as a provider of one or more Service(s)."

http://sdo-phobos.appspot.com/Attorney: "Professional service: Attorney.  This type is deprecated - LegalService is more inclusive and less ambiguous."

Does this address things adequately? The Attorney type remains usable but we encourage people to use LegalService instead.

**Comment by DDeering** [comment]
Dan, I like the use of LegalService for all legal entities such as law firms and notaries, etc.  But one thing I've always struggled with is how to define the type of litigation a law firm offers.  For example, a law firm that offers personal injury representation or automobile accident litigation.  Would it be practical to add a new property and type specifically for LegalService litigation types?  And one thing to keep in mind is that law firms typically offer several types of litigation and legal representation, each usually described on their own page.

My idea may or may not be the best solution, but I do see a need for extending the LegalService type (or Offer) in some way to help clarify what exactly a law firm offers and does.

**Comment by mfhepp** [comment]
@DDeering Note that you can always attach a schema:Offer with a simple name/description for indicating the services offered by a LocalBusiness, so there is no need to provide extra properties or a very deep taxonomy for those IMO.

It's as simple as this:

    <div itemscope itemtype="http://schema.org/Attorney">
     <div itemprop="name">Peter Miller</div>
     <div itemprop="makesOffer" itemscope itemtype="http://schema.org/Offer">
        <div itemprop="name"Automobile Accident Litigation</div>
        <div itemprop="description">Hurt in a car accident? We will help you settle your claims.</div>
        <link itemprop="businessFunction" href="http://purl.org/goodrelations/v1#ProvideService" />
     </div>
    </div>

For two or more services offered, simply use multiple schema:Offer nodes, like so:

    <div itemscope itemtype="http://schema.org/Attorney">
     <div itemprop="name">Peter Miller</div>
     <div itemprop="makesOffer" itemscope itemtype="http://schema.org/Offer">
        <div itemprop="name"Automobile Accident Litigation</div>
        <div itemprop="description">Hurt in a car accident? We will help you settle your claims.</div>
        <link itemprop="businessFunction" href="http://purl.org/goodrelations/v1#ProvideService" />
     </div>
     <div itemprop="makesOffer" itemscope itemtype="http://schema.org/Offer">
        <div itemprop="name"Personal Injury Representation</div>
        <div itemprop="description"> etc. </div>
        <link itemprop="businessFunction" href="http://purl.org/goodrelations/v1#ProvideService" />
     </div>
    </div>

Let's not reinvent the wheel ;-)

**Comment by DDeering** [comment]
Thanks Martin.  That's exactly what I've done up to this point.  It just seems like it's lacking somewhat to me, especially compared to say a MedicalEntity markup.  You can get very granular with MedicalEntity; not so with Attorney or LegalService.

**Comment by vholland** [comment]
@DDeering Do you know of a good taxonomy of practice areas? I think that is what is missing @mfhepp's example: a way to strucuture "automobile accident litigation" such that I know "ambulance chaser" is the same thing.

**Comment by mfhepp** [comment]
@DDeering @vholland if there are external taxonomies, you can use http://schema.org/category on Offer.

As for "especially compared to say a MedicalEntity markup. You can get very granular with MedicalEntity; not so with Attorney or LegalService.": I think that is a bug in the old medical contribution, not a generally working pattern that we should apply to other areas.

In my understanding we are in the process of trimming down the old medical terminology part of schema.org to a more reasonable subset. 

As said, I think Offer + name/category/description is perfectly fine. If you want to be more specific, add the product or service and even add additionalType.

**Comment by mfhepp** [comment]
P.S. Note that you can directly use any of the ca. 3000 entries from the Google taxonomy at

     http://www.google.com/basepages/producttype/taxonomy.en-US.txt

with http://schema.org/category.

There are very few services entries in there at the moment, though,

**Comment by DDeering** [comment]
@vholland I'm not really aware of any external taxonomies related to the legal field, but I'll try to do some digging.

Thanks for the link to Google's taxonomies, @mfhepp.  Unfortunately, currently none are related to law, legal, or attorney.

**Comment by fosterinteractive** [comment]
@danbri - http://sdo-phobos.appspot.com/LegalService Covers all the use cases I've come across. 

Do you have any recommendations as to changing this on live client's sites? Should I wait a while or go ahead and change this quickly (or should this be done with some way of stating "same as" to Attorney)?

**Comment by scor** [comment]
Let's see what the others on the team think, but one idea would be to have both type for a while until any service making use of the old Attorney gets updated. Since you are using RDFa, this is very easy, just add the new type in the typeof attribute with the deprecated type:
```
typeof="schema:LegalService schema:Attorney"
```

Does that sounds like a good approach? The old type could be removed in a few months.

**Comment by danbri** [comment]
@fosterinteractive - please don't stress about updating client sites. Anything marked up as an Attorney is implicitly now considered a LegalService. We don't want to make our improvements create work for you! /cc @scor 

Search engines are generally capable of keeping track of such things. We'll probably make Attorney (and other 'deprecated' types) less prominent within schema.org, to discourage new implementations from adopting it. But we have always kept older terminology documented on the site...

**Comment by scor** [comment]
@danbri Attorney isn't depcrecated to the point of getting its own supersededBy statement in schema.rdfa? Is it purposely a soft-deprecation?

**Comment by scor** [comment]
@danbri found a typo at http://sdo-phobos.appspot.com/ProfessionalService: 
> LegalBusiness was introduced as a more inclusive supertype of Attorney.
should be LegalService. that was the only instance of LegalBusiness I found in the sdo-phobos branch.

**Comment by danbri** [comment]
Thanks @scor - fixed!

**Comment by danbri** [comment]
I am going to close this issue, as I believe the primary point is addressed by the move from http://schema.org/Attorney to http://schema.org/LegalService. We can continue to discuss granularity/precision here, but in the interests of having < 300 open issues I would like to declare that the main problem has been addressed.

**Comment by platinumseo** [comment]
Outstanding resource that has just answered a few of my questions that I've been searching on for a while for a new client.

One question I do have in regard to listing multiple services offered..

Would it be more appropriate to

a) List all the services offered by the legal firm together as touched on in mfhepp's example back on September 29th
b)would it be more appropriate to split them and only include that specific service on it's dedicated page
c)go with A on the primary page/contact/about us pages, and use the Services schema on the dedicated services page?

Thanks again.
